packages/amplication-kafka/src/utils/kafkaEnv.ts

```diff
@@ -0,0 +1,30 @@
+import { EnvironmentVariables } from "./";
+import { KAFKA_BROKERS, KAFKA_CLIENT_ID, KAFKA_GROUP_ID } from "./constants";
+
+export class KafkaEnvironmentVariables extends EnvironmentVariables {
```

Comment:
why do you need the inheritance? you don't inherit anything all the functions are static and you don't have any properties on EnvironmentVariables